src/test/scala/chiseltest/backends/verilator/VerilatorCirctBasicTests.scala

```diff
@@ -0,0 +1,140 @@
+package chiseltest.backends.verilator
+
+import chisel3._
+import chiseltest._
+import chiseltest.simulator.RequiresVerilator
+import chiseltest.tests.{PassthroughModule, StaticModule}
+import firrtl.AnnotationSeq
+import org.scalatest._
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatest.matchers.should.Matchers
+
+class VerilatorCirctBasicTests extends AnyFlatSpec with ChiselScalatestTester with Matchers {
```

Comment:
There should be no need to duplicate the tests here. In general, the compliance tests in simulator should be enough.

If you really want to reuse these tests, I would suggest to make an abstract base class that takes the backend annotation as a constructor argument. Then you can derive both the original Verilator and the VerilatorCirct tests from the same base class. The goal here would be to try and reduce duplicate code.

Reply:
Good point, will delete this file